manifests/7_ingress.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ingress-prombench-9-prometheus-meta
```

Comment:
I accidentally committed prombench-9 instead of prombench-{{.PR_NUMBER}}
Will fix this in next commit

Reply:
git commit --amend :)

Reply:
:grin: